anaconda.spec.in

```diff
@@ -482,6 +482,31 @@ rm -rf \
 %{_prefix}/lib/dracut/modules.d/80%{name}/*
 %{_prefix}/libexec/anaconda/dd_*
 
+# Make sure to not overwrite user configurtion files
```

Comment:
please show the exact rpmlint error that this is fixing. Inline with comment and also in the commit message.
So instead of generic 'Fix rpmlint issues' try to be specific.

Reply:
Done.